tools/cleed_io/slicer.py

```
@@ -0,0 +1,134 @@
+def align_surface(atoms: Atoms, vacuum_direction='z', tolerance=1e-3) -> Atoms:
+    """
+    Aligns the atoms object such that the periodic surface vectors (a, b)
+    lie in the xy-plane (z=0) and the surface normal points along +z.
+    
+    This matches the CLEED convention where z is the surface normal.
+    """
+    # 1. Identify the non-periodic direction or the longest vector if 3D periodic
+    # For a standard slab, we assume c is the vacuum direction.
+    
+    cell = atoms.get_cell()
+    pbc = atoms.get_pbc()
```

Ruff (0.14.8)

Local variable `pbc` is assigned to but never used

Remove assignment to unused variable `pbc`

(F841)